spec/cluster/package.json

```diff
@@ -2,7 +2,7 @@
   "dependencies": {
     "jest": "^29.4.3",
     "localtunnel": "^2.0.2",
-    "twilio": "../../package"
+    "twilio": "../../"
```

Comment:
This is by-design as the cluster tests are intended to be run against the packed output: https://github.com/twilio/twilio-node/blob/4ab01b77da440fe8c711bb12fec0016f1d354069/.github/workflows/test-and-deploy.yml#L44-L49

Reply:
I see. Thanks for the clarification. Closing this PR.